### PR TITLE
PAINTROID-673 : Refactor FillCommandTest to kotlin

### PR DESCRIPTION
**Pull request by Rd4dev**
Jira Ticket: [PAINTROID-673](https://[jira.catrob.at/browse/PAINTROID-673](https://jira.catrob.at/browse/PAINTROID-673))
Description: Refactored FillCommandTest from java to kotlin

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Paintroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [ ] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#paintroid* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Comment by Rd4dev** [comment]
**TODO:** To remove `!! operator`

File: Jenkinsfile
Comment (Lenkomotive) [inline Jenkinsfile:1]: Is there a reason for this jenkinsfile beeing in your PR? If not, please remove it and only include the files you worked on. :)

**Review by Lenkomotive — CHANGES_REQUESTED** [review]

File: Jenkinsfile
Comment (Rd4dev) [reply Jenkinsfile:1]: The Jenkinsfile was inadvertently included in the pull request. 
I had updated the PR to ensure that only the files I directly worked on are included.

**Review by Lenkomotive — APPROVED** [review]
looks good 👍🏼

**Comment by aman021202** [comment]
kindly assign me this issue .

**Review by JulianJautz — APPROVED** [review]
Looks good to me

**Pull request merged**